contracts/deprecated/MerkleDistributor.sol

```diff
@@ -32,30 +36,45 @@ contract MerkleDistributor is Owned, Pausable, IMerkleDistributor {
     function _setClaimed(uint256 index) private {
         uint256 claimedWordIndex = index / 256;
         uint256 claimedBitIndex = index % 256;
-        claimedBitMap[claimedWordIndex] = claimedBitMap[claimedWordIndex] | (1 << claimedBitIndex);
+        claimedBitMap[claimedWordIndex] =
+            claimedBitMap[claimedWordIndex] |
+            (1 << claimedBitIndex);
     }
 
-    function claim(uint256 index, address account, uint256 amount, bytes32[] calldata merkleProof) external override {
+    function claim(
+        uint256 index,
+        address account,
+        uint256 amount,
+        bytes32[] calldata merkleProof
+    ) external override {
         require(!isClaimed(index), "MerkleDistributor: Drop already claimed.");
 
         // Verify the merkle proof.
-        bytes32 node = keccak256(bytes.concat(keccak256(abi.encode(index, account, amount))));
+        bytes32 node = keccak256(
+            bytes.concat(keccak256(abi.encode(index, account, amount)))
+        );
 
-        require(MerkleProof.verify(merkleProof, merkleRoot, node), "MerkleDistributor: Invalid proof.");
+        require(
+            MerkleProof.verify(merkleProof, merkleRoot, node),
+            "MerkleDistributor: Invalid proof."
+        );
 
         // Mark it claimed and send the token.
         _setClaimed(index);
-        require(IERC20(token).transfer(account, amount), "MerkleDistributor: Transfer failed.");
+        require(
+            IERC20(token).transfer(account, amount),
+            "MerkleDistributor: Transfer failed."
+        );
 
         emit Claimed(index, account, amount);
     }
 
-    function _selfDestruct(address payable beneficiary) external onlyOwner {
```

Comment:
@0xlinus just commenting `selfdestruct` out because it's deprecated. We don't use this contract anyway